src/libstore/download.cc

```diff
@@ -230,6 +230,11 @@ struct CurlDownloader : public Downloader
                 curl_easy_setopt(req, CURLOPT_SSL_VERIFYHOST, 0);
             }
 
+            Path netrcFile = settings.get("netrc-file",
+               (format("%1%/%2%") % settings.nixConfDir % "netrc").str());
+            curl_easy_setopt(req, CURLOPT_NETRC_FILE, netrcFile.c_str());
```

Comment:
I mean you pass a c string that is invalidated after this scope, but `curl_easy_setopt` makes a copy of the string passed to it so it's OK.

Can you add a comment about the non-existence?